backend/index.js

```diff
@@ -0,0 +1,50 @@
+require('dotenv').config();
```

Comment:
Regarding putting a bunch of your server code in the backend/index.js. Doing it this way will make writing tests for it more difficult later on during development. A way to make this easier would be to initialize the server and the listener in index.js, but have the rest of your code in a separate file, like server.js or other identifiable name.